docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -2422,6 +2502,95 @@ Display system-wide information
 -   **200** – no error
 -   **500** – server error
 
+### Show docker data usage information
+
+`GET /system/df`
+
+Return docker data usage information
+
+**Example request**:
+
+    GET /system/df HTTP/1.1
+
+**Example response**:
+
+	{
+	    "LayersSize": 1092588,
+	    "Images": [
+	        {
+	            "Id": "sha256:2b8fd9751c4c0f5dd266fcae00707e67a2545ef34f9a29354585f93dac906749",
+	            "ParentId": "",
+	            "RepoTags": [
+	                "busybox:latest"
+	            ],
+	            "RepoDigests": [
+	                "busybox@sha256:a59906e33509d14c036c8678d687bd4eec81ed7c4b8ce907b888c607f6a1e0e6"
+	            ],
+	            "Created": 1466724217,
+	            "Size": 1092588,
+	            "SharedSize": 0,
+	            "VirtualSize": 1092588,
+	            "Labels": {},
+	            "Containers": 1
+	        }
+	    ],
+	    "Containers": [
+	        {
+	            "Id": "e575172ed11dc01bfce087fb27bee502db149e1a0fad7c296ad300bbff178148",
+	            "Names": [
+	                "/top"
+	            ],
+	            "Image": "busybox",
+	            "ImageID": "sha256:2b8fd9751c4c0f5dd266fcae00707e67a2545ef34f9a29354585f93dac906749",
+	            "Command": "top",
+	            "Created": 1472592424,
+	            "Ports": [],
+	            "SizeRootFs": 1092588,
+	            "Labels": {},
+	            "State": "exited",
+	            "Status": "Exited (0) 56 minutes ago",
+	            "HostConfig": {
+	                "NetworkMode": "default"
+	            },
+	            "NetworkSettings": {
```

Comment:
In verbose mode it will eventually be used when we add support for `--format` same as `docker  ps`